src/main/java/frc/robot/subsystems/drive/DriveSubsystem.java

```diff
@@ -477,6 +477,22 @@ public void throttle(double throttleValue) {
     throttleMultiplier = throttleValue;
   }
 
+  public void zeroFrontLeftAtCurrentPos() {
+    Cal.SwerveSubsystem.SWERVE_FRONT_LEFT_ANGULAR_OFFSET_RAD = frontLeft.getEncoderAbsPositionRad();
+  }
+
+  public void zeroFrontRightAtCurrentPos() {
+    Cal.SwerveSubsystem.SWERVE_FRONT_RIGHT_ANGULAR_OFFSET_RAD = frontRight.getEncoderAbsPositionRad();
+  }
+
+  public void zeroRearLeftAtCurrentPos() {
+    Cal.SwerveSubsystem.SWERVE_BACK_LEFT_ANGULAR_OFFSET_RAD = rearLeft.getEncoderAbsPositionRad();
+  }
+
+  public void zeroRearRightAtCurrentPos() {
+    Cal.SwerveSubsystem.SWERVE_BACK_RIGHT_ANGULAR_OFFSET_RAD = rearRight.getEncoderAbsPositionRad();
```

Comment:
For all of these, can you add a System.out.println for what was zeroed and what the value is